front/app/containers/Admin/projects/project/information/ReportSection.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+// api
+import usePhase from 'api/phases/usePhase';
+import useAddReport from 'api/reports/useAddReport';
+import useReport from 'api/reports/useReport';
+import useFeatureFlag from 'hooks/useFeatureFlag';
+
+// components
+import { SectionField, SubSectionTitle } from 'components/admin/Section';
+import { Box, Text } from '@citizenlab/cl2-component-library';
+import Button from 'components/UI/Button';
+import ReportRow from './ReportRow';
+
+// i18n
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from './messages';
+
+interface Props {
+  phaseId: string;
+}
+
+const ReportSection = ({ phaseId }: Props) => {
+  const { data: phase } = usePhase(phaseId);
+  const { mutate: createReport, isLoading } = useAddReport();
+  const phaseReportsEnabled = useFeatureFlag({ name: 'phase_reports' });
+
+  const reportId = phase?.data.relationships.report?.data?.id;
+  const { data: report } = useReport(reportId);
+
+  if (phase === undefined || !phaseReportsEnabled) return null;
+
+  const handleCreateReport = () => {
+    createReport({ phase_id: phaseId });
+  };
+
+  return (
+    <SectionField className="fullWidth">
```

Comment:
This section is quite different when the report is created and when it's not created yet. Maybe it's a sign that it should be two separate components.

Reply:
I think it's ok. If I split it up in two different components, I have to either move the report fetching and everything into the already way too big `edit` parent component, or I have to create a third component that does that and then renders one of the two children based on if there is a report or not. Also seems overkill for something so simple